docs/user/installation_and_configuration.rst

```diff
@@ -1,17 +0,0 @@
-Installation and Configuration
```

Comment:
I'm all for getting rid of the installation section, but I think there should still be a short chapter near the beginning of the docs called ``installation.rst`` that just says the installation instructions are broken up by backend and then provides links to each.